petab/visualize/data_overview.py

```diff
@@ -7,6 +7,7 @@
 
 import pandas as pd
 import petab
+from petab.C import *
```

Comment:
maybe ``# noqa: F403``

Reply:
Seems to be redundant with flake `--extend-ignore=F403`